### ISSUE TITLE
Flashing z-stack examples fail with error "Timeout waiting for ACK/NACK after 'Get Status (0x23)'"

### ISSUE THREAD

**Issue by Caian** [JelmerT/cc2538-bsl#112]
I'm having the following error when trying to flash z-stack examples that I compiled using IAR:

```
Writing 524256 bytes starting at address 0x00200000
Write 232 bytes at 0x0027FEF88
ERROR: Timeout waiting for ACK/NACK after 'Get Status (0x23)'
```

On the other hand, firmwares that I find online, like this one: https://github.com/jethome-ru/zigbee-firmware flash just fine. I've tried `cc2538-bsl.py -p /dev/serial0 -w <firmware>` and `cc2538-bsl.py -p /dev/serial0 -e -v -w <firmware>` with both raw binaries and intel hex. The result is always the same.

I'm using a cheap cc2538 + cc2592 board that I wired to a raspberry pi and I'm using openocd to reset it back to bootloader mode.

**Comment by JelmerT** [comment]
That's an odd place for it to error out. What I'm suspecting is that you're writing to a memory address that the chip is not very happy with and then stops responding.
Does your firmware work if you flash it via jtag?

**Comment by Caian** [comment]
Yes, I've managed to find a custom build of OpenOCD that flashes and verifies it just fine. I've also tried to reduce the baud to 9600 with no success.

I tested basic stuff in the program, like simple IO and UART, but I can't fully confirm that the flash was a success because it was meant to be used with a developer board.

**Comment by jeroenvdd** [comment]
I now get exactly the same unfortunatily
